backend/app/tests/api/routes/test_users.py

```diff
@@ -182,9 +184,18 @@ def test_update_user_me(
     assert updated_user["email"] == email
     assert updated_user["full_name"] == full_name
 
+    # * db asserts
+    user_query = select(User).where(User.email == email)
+    user_db = db.exec(user_query).first()
+    db.refresh(user_db)
+    if not user_db:
+        pytest.fail("user not found")
```

Comment:
```suggestion
```

We can probably remove these, the asserts below would ensure the `user_db` exists.

Reply:
Yeah, that's the solution to avoid mypy asking for it just doing assert user_db. It's done now